src/schema/home/fetch.js

```diff
@@ -13,8 +11,8 @@ import {
 } from "lodash"
 import blacklist from "lib/artist_blacklist"
 
-export const featuredFair = () => {
-  return gravity("fairs", {
+export const featuredFair = fairsLoader => {
```

Comment:
Here are helper methods that actually fetch, now accepting their specific corresponding loader.